src/work-allocation/guards/tribunal-caseworker-guard.ts

```diff
@@ -16,8 +15,7 @@ export class TribunalCaseworkerGuard implements CanActivate {
         const userInfoStr = this.sessionStorageService.getItem('userDetails');
         if (userInfoStr) {
             const userInfo: UserInfo = JSON.parse(userInfoStr);
-            isActivate = userInfo.roles.includes(TribunalCaseworkerGuard.caseWorkerRole) ||
-            userInfo.roles.includes(TribunalCaseworkerGuard.seniorCaseWorkerRole);
+            isActivate = userInfo.roles.includes(TribunalCaseworkerGuard.caseWorkerRole);
```

Comment:
Is there any unit tests(changes of unit test) to cover this?

Reply:
@rayliangatos yes please